src/modules/deposit/service.ts

```diff
@@ -36,6 +38,44 @@ export class DepositService {
     return data;
   }
 
+  public async getUserDeposits(userAddress: string, status?: "filled" | "pending", limit = 10, offset = 0) {
+    let userDeposits: Deposit[] = [];
+    let total = 0;
+
+    if (!utils.isAddress(userAddress)) {
```

Comment:
The addresses are stored in the DB in checksum form, so let's transform the `userAddress` by calling `ethers.utils.getAddress()` in a `try catch` and throw `InvalidAddressException` if `getAddress()` fails

Reply:
Changed it [here](https://github.com/across-protocol/scraper-api/pull/182/commits/0dade652216858a0130582b1c0cdd3975942c3a3)